### PR TITLE
Remove static from ForgeConfig

### PR DESCRIPTION
**Pull request by igor-aptos**
Currently, all tests need to be statically defined, which precludes using any utility function for their creation, creating dynamic objects (like vectors), etc. Boxing everything, resolves this issue

### CODE REVIEW

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -464,34 +464,35 @@ fn get_test_suite(suite_name: &str, duration: Duration) -> Result<ForgeConfig<'s
...
 }
 
-fn local_test_suite() -> ForgeConfig<'static> {
+fn local_test_suite() -> ForgeConfig {
     ForgeConfig::default()
-        .with_aptos_tests(vec![&FundAccount, &TransferCoins])
-        .with_admin_tests(vec![&GetMetadata])
-        .with_network_tests(vec![&RestartValidator, &EmitTransaction])
+        .with_aptos_test(FundAccount)
+        .with_aptos_test(TransferCoins)
```
Comment (ibalajiarun) [inline testsuite/forge-cli/src/main.rs:477]: nit: Seems confusing that chaining actually appends. Usually I would think thee last call should overwrite the previous ones?

may be use `append_network_test` or a better name?

**Review by perryjrandall — APPROVED** [review]
Very clean

**Review by ibalajiarun — APPROVED** [review]

File: crates/transaction-emitter-lib/src/emitter/stats.rs [hunk 2]
```diff
@@ -40,8 +40,12 @@ impl fmt::Display for TxnStatsRate {
...
         write!(
             f,
-            "submitted: {} txn/s, committed: {} txn/s, expired: {} txn/s, failed submission: {} tnx/s, latency: {} ms, (p50: {} ms, p90: {} ms, p99: {} ms), latency samples: {}",
-            self.submitted, self.committed, self.expired, self.failed_submission, self.latency, self.p50_latency, self.p90_latency, self.p99_latency, self.latency_samples,
+            "committed: {} txn/s{}{}{}, latency: {} ms, (p50: {} ms, p90: {} ms, p99: {} ms), latency samples: {}",
+            self.committed,
+            if self.submitted != self.committed { format!(", submitted: {} txn/s", self.submitted) } else { "".to_string()},
+            if self.failed_submission != 0 { format!(", failed submission: {} txn/s", self.failed_submission) } else { "".to_string()},
+            if self.expired != 0 { format!(", expired: {} txn/s", self.expired) } else { "".to_string()},
+            self.latency, self.p50_latency, self.p90_latency, self.p99_latency, self.latency_samples,
```
[the remark this answers is not in the document]
Comment (igor-aptos) [reply crates/transaction-emitter-lib/src/emitter/stats.rs]: this change is not part of this PR

with github and stacked PRs , it depends when you look (i.e. if I update the bottom PR, but have yet to update this one, both changes will show up here)

**Review by sitalkedia — APPROVED** [review]
LGTM

**Pull request merged**